sitegen/src/main/java/io/helidon/sitegen/Helper.java

```diff
@@ -89,8 +89,7 @@ public static Path loadResourceDirAsPath(String resourcePath)
                 templatesDir = fs.getPath(relativePath);
                 break;
             case "file":
-                templatesDir = FileSystems.getDefault().getPath(
-                        templatesDirURI.getSchemeSpecificPart());
+                templatesDir = new File(templatesDirURI).toPath();
```

Comment:
How about `templatesDir = Paths.get(templatesDirURI);` instead? Shorter, no throwaway `File` instance created.

Reply:
Seems fair, I'll try it